tensorflow_quantum/core/src/util_qsim.h

```diff
@@ -308,7 +308,10 @@ tensorflow::Status AccumulateOperators(
   ss.CopyState(source, scratch);
   ss.SetAllZeros(dest);
 
-  for (int i = 0; i < p_sums.size(); i++) {
+  DCHECK_EQ(p_sums.size(), op_coeffs.size());
+  const int num_p_sums = p_sums.size();
+
+  for (int i = 0; i < num_p_sums; i++) {
```

Comment:
Would `size_t` for `i` work instead ? It seems like that would be the type that `p_sums.size()` returns ?

Reply:
Yes, it would work. I was just trying to follow the Google style guide:
https://google.github.io/styleguide/cppguide.html#Integer_Types

> Because of historical accident, the C++ standard also uses unsigned integers to represent the size of containers - many members of the standards body believe this to be a mistake
> and try to avoid unsigned types (except for representing bitfields or modular arithmetic)

That being said, I am far from a C++ expert and I'm quite flexible on the whole thing.

Reply:
The guide also says using size_t is fine and in our case it'd let us cut down to just 

```C++
for(size_t i =0;i<p_sums.size();i++){
  ...
}
```

Reply:
Done.